src/splitInteger.test.js

```diff
@@ -4,18 +4,20 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+    expect(splitInteger(8, 1)).toEqual([8]);
```

Comment:
The description for this test case seems incorrect. It mentions splitting a number into equal parts if divisible, but the test is actually checking if splitting 8 into 1 part returns [8]. Consider revising the description to accurately reflect the test case.